changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.7.0-beta8": [
+      "[Fixed] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

The PR this was fixed in is #7508.

---

Also, should this be [Added] instead of [Fixed]???

---

ohh yeah those are both good points

---

actually the issue linked there is the original issue
```suggestion
      "[Added] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

---

Yea, I know, figured the PR was better since we are giving someone a shoutout.